### PR TITLE
Add incorrect file

### PR DESCRIPTION
**Pull request by matthieuauger**
Hello

### CODE REVIEW

File: IncorrectFile2.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace AppBundle;
+
+class MyClass
+{
+  function MyFunction()
```
Comment (matthieuauger) [inline IncorrectFile2.php]: Line indented incorrectly; expected 4 spaces, found 2

Comment (matthieuauger) [inline IncorrectFile2.php]: Method name "MyClass::MyFunction" is not in camel caps format

Comment (matthieuauger) [inline IncorrectFile2.php]: Visibility must be declared on method "MyFunction"

File: IncorrectFile2.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    {
```
Comment (matthieuauger) [inline IncorrectFile2.php]: Opening brace indented incorrectly; expected 2 spaces, found 4

File: IncorrectFile2.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+    }
```
Comment (matthieuauger) [inline IncorrectFile2.php]: Closing brace indented incorrectly; expected 2 spaces, found 4

File: IncorrectFile2.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+}
```
Comment (matthieuauger) [inline IncorrectFile2.php]: Expected 1 blank line at end of file; 3 found

File: IncorrectFile2.php (hunk 1 above)
Comment (pull-request-checker) [inline IncorrectFile2.php]: Line indented incorrectly; expected 4 spaces, found 2

Comment (pull-request-checker) [inline IncorrectFile2.php]: Method name "MyClass::MyFunction" is not in camel caps format

Comment (pull-request-checker) [inline IncorrectFile2.php]: Visibility must be declared on method "MyFunction"

File: IncorrectFile2.php (hunk 2 above)
Comment (pull-request-checker) [inline IncorrectFile2.php]: Opening brace indented incorrectly; expected 2 spaces, found 4

File: IncorrectFile2.php (hunk 3 above)
Comment (pull-request-checker) [inline IncorrectFile2.php]: Closing brace indented incorrectly; expected 2 spaces, found 4

File: IncorrectFile2.php (hunk 4 above)
Comment (pull-request-checker) [inline IncorrectFile2.php]: Expected 1 blank line at end of file; 3 found

**Pull request closed without merging**